### PR TITLE
Improving README

### PR DESCRIPTION
**Pull request by wesleyhf**
## README

- moving Environment Variables section to table
- adding COMPOSER_MEMORY_LIMIT on documentation (https://github.com/kool-dev/docker-php/commit/bb4fcc396216fb1587598068c3ad3b0fd1ace1b0)
- adding new NGINX variables (#9)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,19 +52,21 @@ Minimal PHP Docker image focused on Laravel applications. It's use is intended f
...
-- **PHP_MEMORY_LIMIT** - Changes PHP memory limit
-- **PHP_MAX_INPUT_VARS** - Changes how many input variables may be accepted on PHP
-- **PHP_UPLOAD_MAX_FILESIZE** - Changes PHP maximum size of an uploaded file
-- **PHP_POST_MAX_SIZE** - Changes PHP max size of post data allowed
-- **PHP_MAX_EXECUTION_TIME** - Changes PHP maximum time is allowed to run a script
-- **PHP_FPM_LISTEN** - Changes the PORT address of the FastCGI requests
-- **PHP_FPM_MAX_CHILDREN** - Changes the number of child processes to be used on FPM
-- **PHP_FPM_REQUEST_TERMINATE_TIMEOUT** - Changes FPM timeout to serve a single request
-- **PHP_FPM_REQUEST_TERMINATE_TIMEOUT** - Changes FPM timeout to serve a single request
+Variable | Description | Default Value
```
Comment (fabriciojs) [inline README.md:55]: can we change the order to help readability? making it `Variable | Default Value | Description`

**Review by fabriciojs — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -52,19 +52,21 @@ Minimal PHP Docker image focused on Laravel applications. It's use is intended f
...
+**COMPOSER_MEMORY_LIMIT** | Changes composer memory limit | `-1`
+**ENABLE_XDEBUG** | Enables the Xdebug extension **(only prod versions)** | `false`
+**PHP_MEMORY_LIMIT** | Changes PHP memory limit | `256M`
+**PHP_MAX_INPUT_VARS** | Changes how many input variables may be accepted on PHP | `1000` 
+**PHP_UPLOAD_MAX_FILESIZE** | Changes PHP maximum size of an uploaded file | `25M`
+**PHP_POST_MAX_SIZE** | Changes PHP max size of post data allowed | `25M`
+**PHP_MAX_EXECUTION_TIME** | Changes PHP maximum time is allowed to run a script | `30`
+**PHP_FPM_LISTEN** | Changes the PORT address of the FastCGI requests | `9000`
+**PHP_FPM_MAX_CHILDREN** | Changes the number of child processes to be used on FPM | `10`
+**PHP_FPM_REQUEST_TERMINATE_TIMEOUT** | Changes FPM timeout to serve a single request | `60`
```
Comment (dbpolito) [inline README.md:70]: please add another section for NGINX variables...

Comment (dbpolito) [reply README.md:70]: or a way to make it clear that these are for all images and nginx has a few more

**Comment by dbpolito** [comment]
@wesleyhf 

We just introduced a few more env variables on nginx images: https://github.com/kool-dev/docker-php/pull/9

**Review by dbpolito — APPROVED** [review]

**Pull request merged**